src/jagex2/io/Jagfile.test.ts

```diff
@@ -18,17 +18,18 @@ describe('Jagfile', (): void => {
 
     describe('constructor', (): void => {
         it('result after creation', (): void => {
-            const packet: Packet = new Packet(new Uint8Array(18));
+            const packet: Packet = new Packet(new Uint8Array(19));
             packet.p3(1); // unpackedSize
             packet.p3(1); // packedSize
             packet.p2(1); // fileCount
             packet.p4(-1502153170); // hitmarks.dat
             packet.p3(1); // fileUnpackedSize[0]
             packet.p3(1); // filePackedSize[0]
+            packet.p1(255); // hitmarks.dat file data
```

Comment:
```suggestion
            packet.p1(255); // add one byte of data to match `filePackedSize`
```

would recommend to replace this comment because `255` isn't actually the data of `hitmarks.dat`   (the comment above on line 25 is because `-1502153170` is the hash of `hitmarks.dat`

Reply:
Okay, done